bitvision/engine/data_bus.py

```diff
@@ -97,48 +105,55 @@ def collect_articles(urls, end_date, filename):
 
 
 def fetch_price_data():
-    return pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHARTS/\
-                        BITSTAMPUSD.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
+    return pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHARTS/BITSTAMPUSD.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
+
 
 def fetch_blockchain_data():
     # Loads datasets from Quandl
-    conf_time = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                             ATRCT.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    block_size = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                              AVBLS.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    txn_cost = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                            CPTRA.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    difficulty = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                              DIFF.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    txn_count = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                             NTRAN.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    hash_rate = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                             HRATE.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    market_cap = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                              MKTCP.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    miners_rev = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                              MIREV.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    block_txn = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                             NTRBL.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    unique_addr = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                               NADDU.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    total_btc = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                             TOTBC.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
-    txn_fees = pd.read_csv("https://www.quandl.com/api/v3/datasets/BCHAIN/\
-                            TRFUS.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
+    conf_time = pd.read_csv(
+        "https://www.quandl.com/api/v3/datasets/BCHAIN/ATRCT.csv?api_key=iKmHLdjz-ghzaWVKyEfw", sep=',')
```

Comment:
How about moving these URLs to a dict?

```python
quandl = {
    "conf_time": "https://www.quandl.com/api/v3/datasets/BCHAIN/ATRCT.csv?api_key=iKmHLdjz-ghzaWVKyEfw",
    # ...
}

# So now you can do this:

```python
return pd.read_csv(quandl["conf_time"], sep=',')
```